tests/base.py

```diff
@@ -239,5 +242,17 @@ def __init__(self):
         self.template_system = self
         self.name = 'mako'
 
+    def _activate_plugins_of_category(self, category):
```

Comment:
Nah, let’s just keep it duplicated.  Merging.

(Invariance failures are caused by this being branched out before rst.css and aria fixes.)